README.md

```diff
@@ -25,3 +27,31 @@ cookiecutter mbuild-cookiecutter
 
 From here, the user will be prompted for information on how to build
 the directory such as package name, module name, and authors.
+
+## Testing
+
+Unit testing with [pytest](https://pytest.org) has been integrated
+within this cookiecutter.  This testing framework is used 
+in [mBuild](https://github.com/mosdef-hub/mbuild).  The default tests
+added in `project/tests/` are to check that the cookiecutter package
```

Comment:
```suggestion
added in `{{cookiecutter.directory_name}}/tests/` are to check that the cookiecutter package
```

Not sure if this would be more clear or potentially confusing (another one) below)